src/testServer.js

```diff
@@ -0,0 +1,36 @@
+import Promise from 'bluebird'
+import child_process from 'child_process'
+import process from 'process'
+import path from 'path'
+
+/**
+ * Start a test server
+ */
+
+let _child, waiting
+
+export default function testServer(script="index.js", env={}) {
+  if (_child) {
```

Comment:
I don't understand the use of `_child` here. I would think you would want to always return the `waiting` Promise (whether it's resolved or not). That way, `testServer()` always returns a promise (rather than sometimes a promise, sometimes a `ChildProcess`) so it's safely then-able.